src/Employer/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -1,31 +1,46 @@
 using Esfa.Recruit.Employer.Web.Configuration;
 using Esfa.Recruit.Employer.Web.Configuration.Routing;
 using Esfa.Recruit.Employer.Web.Orchestrators;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
+using System;
 using System.Threading.Tasks;
+using Esfa.Recruit.Shared.Web.Extensions;
+using Microsoft.AspNetCore.Hosting;
 
 namespace Esfa.Recruit.Employer.Web.Controllers
 {
     [Route(RoutePrefixPaths.AccountRoutePath)]
     public class DashboardController : Controller
     {
+        private readonly IHostingEnvironment _hostingEnvironment;
         private readonly DashboardOrchestrator _orchestrator;
 
-        public DashboardController(DashboardOrchestrator orchestrator)
+        public DashboardController(IHostingEnvironment hostingEnvironment, DashboardOrchestrator orchestrator)
         {
+            _hostingEnvironment = hostingEnvironment;
             _orchestrator = orchestrator;
         }
 
         [HttpGet("", Name = RouteNames.Dashboard_Index_Get)]
-        public async Task<IActionResult> Dashboard([FromRoute]string employerAccountId)
+        public async Task<IActionResult> Dashboard([FromRoute]string employerAccountId, [FromQuery]string statusFilter)
         {
             var vm = await _orchestrator.GetDashboardViewModelAsync(employerAccountId);
             
             if (TempData.ContainsKey(TempDataKeys.DashboardErrorMessage))
             {
                 vm.WarningMessage = TempData[TempDataKeys.DashboardErrorMessage].ToString();
             }
+            
+            if (!string.IsNullOrEmpty(statusFilter) && Enum.TryParse<VacancyStatus>(statusFilter, out var filter))
```

Comment:
Maybe move this if/else into its own method / extension method

Reply:
Have moved into a method.